app/components/views/PKlima/index.tsx

```diff
@@ -36,6 +35,7 @@ interface Props {
   provider?: providers.JsonRpcProvider;
   address?: string;
   isConnected?: boolean;
+  toggleModal?: () => void;
```

Comment:
should always exist, see related suggestions above plz update all the other `views` component props to be non-optional 
```suggestion
  toggleModal: () => void;
```

Reply:
updated